modules/users/client/components/UserSummary.js

```diff
@@ -0,0 +1,43 @@
+// External dependencies
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+// Internal dependencies
+import Avatar from '@/modules/users/client/components/Avatar.component';
+
+export default function UserSummary({ user }) {
+  const UserSummary = styled.div.attrs({
```

Comment:
This caused me to do a double take, a UserSummary inside a UserSummary? I can't think of a better name, maybe just Container? What are the other styled components called?

Reply:
Haha, you're right. I saw `Container` in @mrkvon 's PR as well; seems indeed like a good convention to have.